toqito/channel_metrics/diamond_norm_distance.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 
 
-def diamond_norm(choi_1: np.ndarray, choi_2: np.ndarray) -> float:
+def diamond_norm_distance(choi_1: np.ndarray, choi_2: np.ndarray) -> float:
```

Comment:
Since the `diamond_norm` function is already in the `channel_metrics/` folder, it's already implied that this is a distance metric. I would opt to keep the name `diamond_norm` over `diamond_norm_distance` (same with the file name as well, etc.)